assets/js/base/context/providers/store-notices/components/style.scss

```diff
@@ -22,3 +22,10 @@
 		margin-top: 1em;
 	}
 }
+
+.theme-twentytwentyone,
+.theme-twentytwenty {
+	.wc-block-components-notices__notice {
+		padding: 1.5rem 3rem;
+	}
+}
```

Comment:
Seems like a weird place for this to live, can we add a `@todo` here to move it to WooCommerce Core?